modules/gapi/misc/python/test/test_gapi_infer.py

```diff
@@ -9,23 +9,205 @@
 
 class test_gapi_infer(NewOpenCVTests):
 
-    def test_getAvailableTargets(self):
-        targets = cv.dnn.getAvailableTargets(cv.dnn.DNN_BACKEND_OPENCV)
-        self.assertTrue(cv.dnn.DNN_TARGET_CPU in targets)
+    def infer_reference_network(self, model_path, weights_path, img):
+        net = cv.dnn.readNetFromModelOptimizer(model_path, weights_path)
+        net.setPreferableBackend(cv.dnn.DNN_BACKEND_INFERENCE_ENGINE)
+        net.setPreferableTarget(cv.dnn.DNN_TARGET_CPU)
+
+        blob = cv.dnn.blobFromImage(img)
+
+        net.setInput(blob)
+        return net.forward(net.getUnconnectedOutLayersNames())
+
+
+    def make_roi(self, img, roi):
+        return img[roi[1]:roi[1] + roi[3], roi[0]:roi[0] + roi[2], ...]
 
 
     def test_age_gender_infer(self):
+        # NB: Check IE
+        if not cv.dnn.DNN_TARGET_CPU in cv.dnn.getAvailableTargets(cv.dnn.DNN_BACKEND_INFERENCE_ENGINE):
+            return
+
+        root_path    = '/omz_intel_models/intel/age-gender-recognition-retail-0013/FP32/age-gender-recognition-retail-0013'
```

Comment:
Don't we have some policy regarding our tests, that if we don't find models/data, we don't run the tests? Won't those tests start failing later with new paths?

Reply:
I believe we don't care about it in these tests.